Source/Core/InputCommon/ControllerInterface/DInput/DInputKeyboardMouse.h

```diff
@@ -106,11 +116,36 @@ class KeyboardMouse : public Core::Device
 
 private:
   void UpdateCursorInput();
+  void Generate_Octagon_Points(POINT octagon_points[8]);
+  bool Point_Is_Inside_Octagon(const POINT& mouse_point, const POINT octagon_points[8]);
```

Comment:
```suggestion
  bool Point_Is_Inside_Octagon(const POINT& mouse_point, const POINT octagon_points[8]) const;
```

or it can be made static if it doesn't depend on class state (if that's the case, then the function can be fully moved into the cpp file). Ditto for the following two functions as well

Reply:
Yeah, there are a bunch of helper functions that I put in the private section that I could definitely put into the .cpp file. Would you prefer they were taken out of the DInputKeyboardMouse class entirely? What I'm hearing from your comment is that having them as static functions outside of the class would be most appropriate here.

Reply:
Yeah, there's no need to expose them in the overall class interface if they're just internal helper functions